### PR TITLE
Add docs for `Complex#zero?`

### PR DESCRIPTION
**Pull request by wontruefree**

### CODE REVIEW

File: src/complex.cr [hunk 1]
```diff
@@ -251,6 +251,16 @@ struct Complex
     new 0, 0
   end
 
+  # Returns a `true` if the complex number is zero.  This means the real and imaginary are both zero.
```
Comment (straight-shoota) [inline src/complex.cr:254]:
```suggestion
  # Returns `true` if the complex number is zero.  This means the real and imaginary are both zero.
```

File: src/complex.cr [hunk 2]
```diff
@@ -251,6 +251,16 @@ struct Complex
...
+  #
+  # ```
+  # require "complex"
+  #
+  # Complex.new(0, 0).zero? # => true
+  # Complex.new(1, 0).zero? # => false
+  # Complex.new(0, 1).zero? # => false
+  #
```
Comment (straight-shoota) [inline src/complex.cr:262]:
```suggestion
```

**Review by straight-shoota — APPROVED** [review]

File: src/complex.cr [hunk 3]
```diff
@@ -251,6 +251,15 @@ struct Complex
     new 0, 0
   end
 
+  # Returns `true` if the complex number is zero.  This means the real and imaginary are both zero.
```
Comment (Sija) [inline src/complex.cr:254]:
```suggestion
  # Returns `true` if the complex number is zero, 
  # This means the real and imaginary are both zero.
```

Comment (straight-shoota) [reply src/complex.cr:254]: Comma?

Comment (Sija) [reply src/complex.cr:254]: Newline, and yes, comma.

Comment (straight-shoota) [reply src/complex.cr:254]: That combination doesn't make sense to me. It's either comma *or* newline.

But these are really two separate sentences. Concatenating them does no good. So no comma, please. Newline is fine.

**Review by beta-ziliani — APPROVED** [review]
Thanks Jack!

**Review by straight-shoota — APPROVED** [review]

**Pull request merged**